js/script.js

```diff
@@ -0,0 +1,38 @@
+var tweetLink = "https://twitter.com/intent/tweet?text=";
+var url = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+
+document.addEventListener('DOMContentLoaded', function() {
+  getData();
+  document.querySelector('.container__button')
+  .addEventListener('click', function() {
+    getData();
+  });
+});
+
+var prepareTweet = function(input) {
+  var data = input[0];
```

Comment:
from where you know that input is an array, before you assign first element of input check that input is an array :)